### PR TITLE
Setup GraphQL

### PR DESCRIPTION
**Pull request by aJerryGuo**
## Description

1. graphql-playground at `/graphiql`
2. GraphQL API endpoint at `/api`
3. The schema of award may be changed in the future.

## TODOs

1. implement `list_awards/0` test

### CODE REVIEW

**Review by aMikeChen — APPROVED** [review]

File: lucky_draw/lib/lucky_draw_web/schema.ex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+defmodule LuckyDrawWeb.Schema do
+  use Absinthe.Schema
+  import_types(LuckyDrawWeb.Schema.Lottery)
+
+  alias LuckyDrawWeb.Resolver
+
+  query do
+    @desc "Get all awards"
+    field :awards, list_of(:award) do
+      resolve(&Resolver.Lottery.list_awards/3)
```
Comment (aStanleyLiang) [inline lucky_draw/lib/lucky_draw_web/schema.ex:10]: learned something, cool
_reference to the 3 arity function all_links found in the CommunityWeb.NewsResolver module_

originally this is weird to me til I search for it.

File: lucky_draw/priv/repo/seeds.exs [hunk 2]
```diff
@@ -1,11 +1,22 @@
...
+alias LuckyDraw.Lottery.Award
+
+opts = [on_conflict: :replace_all, conflict_target: :id]
+
+Repo.insert!(%Award{id: 1, content: "The Last of Us II", price: 590}, opts)
+
+Repo.insert!(
+  %Award{
+    id: 2,
+    content: "PS5 Limited Edition The Last of Us II GOTY Bundle",
```
Comment (aStanleyLiang) [inline lucky_draw/priv/repo/seeds.exs]: I cannot believe you are fans of 10/10 🙈

I would like to promote cyberpunk 2077 to u despite it's argued by lots of bugs now

**Review by aStanleyLiang — APPROVED** [review]
LGTM

File: lucky_draw/priv/repo/seeds.exs (hunk 2 above)
Comment (aJerryGuo) [reply lucky_draw/priv/repo/seeds.exs]: cuz I'm well educated! must play and you don't want to miss!

**Pull request merged**